content/en/includes/armory-operator/operator-compat-matrix.md

```diff
@@ -1,6 +1,6 @@
-| Kubernetes Version         | Armory Operator Version        | Armory CD Version         |
-| :------------------------- | :----------------------------- | :-------------------------------- |
-| < 1.21                     | <= 1.6.x                       | <= 2.28.0                         |
-| >= 1.21                    | >= 1.7.x                       | All Supported Continuous Deployment Versions |
+| Kubernetes Version         | Armory Operator Version        | Spinnaker Operator Version     | Armory CD Version         |
+| :------------------------- | :----------------------------- | :----------------------------- | :------------------------ |
+| < 1.21                     | <= 1.6.x                       | <= 1.2.x                       | <= 2.28.0                 |
+| >= 1.21                    | >= 1.7.x                       | >= 1.3.x                       | All Supported Continuous Deployment Versions |
```

Comment:
If we go with the >= approach that means we need to ensure that when a new version of Kubernetes comes out we have to ensure 1.3.0 continues to work.  Or should we be explicit and just say 1.22 to 1.24?